README.md

```
@@ -78,7 +78,7 @@ Quill is a free Mac dictation app inspired by [Wispr Flow](https://wisprflow.ai/
 
 ## Quick Start
 
-1. Download the app from above or [click here](https://github.com/woosublee/freeflow/releases/latest/download/Quill.dmg)
+1. Download the app from above or [click here](https://github.com/woosublee/quill/releases/latest/download/Quill.dmg)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)